packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## NEXT
+## 0.6.13+1
 
-* Updates the example to specify the import for `DropdownMenu`.
```

Comment:
nit: Did you mean to delete this?

Reply:
Oops! I guess I'm out of practice at changing changelogs manually...